hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java

```diff
@@ -77,9 +77,11 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
 
         final var transactionBody = context.body().fileAppendOrThrow();
         final var fileStore = context.createStore(ReadableFileStore.class);
-        preValidate(transactionBody.fileID(), fileStore, context, false);
+        final var transactionFileId = transactionBody.fileID();
+        requireNonNull(transactionFileId);
```

Comment:
These lines can be combined.
```suggestion
        final var transactionFileId = requireNonNull(transactionBody.fileID());
```

Reply:
fixed thanks